bigtable/admin.go

```diff
@@ -1119,19 +1113,27 @@ type ClusterConfig struct {
 
 	// AutoscalingConfig configures the autoscaling properties on a cluster.
 	// One of NumNodes or AutoscalingConfig is required.
-	AutoscalingConfig AutoscalingConfig
+	AutoscalingConfig *AutoscalingConfig
 }
 
 func (cc *ClusterConfig) proto(project string) *btapb.Cluster {
-	return &btapb.Cluster{
+	cl := &btapb.Cluster{
 		ServeNodes:         cc.NumNodes,
 		DefaultStorageType: cc.StorageType.proto(),
 		Location:           "projects/" + project + "/locations/" + cc.Zone,
 		EncryptionConfig: &btapb.Cluster_EncryptionConfig{
 			KmsKeyName: cc.KMSKeyName,
 		},
-		Config: cc.AutoscalingConfig.proto(),
 	}
+
+	if asc := cc.AutoscalingConfig.proto(); asc != nil {
+		cl.Config = &btapb.Cluster_ClusterConfig_{
+			ClusterConfig: &btapb.Cluster_ClusterConfig{
+				ClusterAutoscalingConfig: cc.AutoscalingConfig.proto(),
```

Comment:
You can just set it equal to `asc` here correct?

Reply:
Woops. Yes.

Reply:
Actually, I removed the `return nil` statement inside `proto` and cleaned this up as well.